### PR TITLE
Resolve EA logging issue + Accessibility on Widget List

### PR DESCRIPTION
**Pull request by bordoni**
None of the changes are complex, resolving two issues customers were facing.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -225,6 +225,8 @@ Remember to always make a backup of your database and files before updating!
 
 * Feature - Add new subscribe options to the "Export Events" link at the bottom of calendar views and single Event view [TEC-4131]
 * Fix - Bypass caching of the event when dealing with the admin event list table. [TEC-4156]
+* Fix - Event List Widget will not print the H3 tag related to the title when that is empty.
```
Comment (EugeneKyale) [inline readme.txt:228]: We need to have the ticket codes for these changes for future tracking.

File: readme.txt [hunk 2]
```diff
@@ -225,6 +225,8 @@ Remember to always make a backup of your database and files before updating!
...
+* Tweak - Allow bypassing of the Aggregator logging function by using the `tec_aggregator_records_should_log_error` filter.
```
Comment (EugeneKyale) [inline readme.txt:229]: Same comment about the ticket codes.

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 3]
```diff
@@ -425,9 +426,9 @@ public function dont_change_post_modified( $data, $postarr ) {
 	/**
 	 * Preps post arguments for create/save
 	 *
-	 * @param string  $type Type of record to create - manual or schedule
-	 * @param object  $args Post type args
-	 * @param array   $meta Post meta
+	 * @param string $type Type of record to create - manual or schedule
```
Comment (EugeneKyale) [inline src/Tribe/Aggregator/Record/Abstract.php:429]:
```suggestion
	 * @param string $type Type of record to create - manual or schedule.
```

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 4]
```diff
@@ -425,9 +426,9 @@ public function dont_change_post_modified( $data, $postarr ) {
...
+	 * @param object $args Post type args
```
Comment (EugeneKyale) [inline src/Tribe/Aggregator/Record/Abstract.php:430]:
```suggestion
	 * @param object $args Post type args.
```

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 5]
```diff
@@ -425,9 +426,9 @@ public function dont_change_post_modified( $data, $postarr ) {
...
+	 * @param array  $meta Post meta
```
Comment (EugeneKyale) [inline src/Tribe/Aggregator/Record/Abstract.php:431]:
```suggestion
	 * @param array  $meta Post meta.
```

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 6]
```diff
@@ -645,7 +646,7 @@ public function create_child_record() {
 	 * @param array $meta Meta to add to the post
```
Comment (EugeneKyale) [inline src/Tribe/Aggregator/Record/Abstract.php:646]:
```suggestion
	 * @param array $meta Meta to add to the post.
```

**Comment by EugeneKyale** [comment]
Lets also make sure the tests are passing.

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 7]
```diff
@@ -645,7 +646,7 @@ public function create_child_record() {
...
 	 */
 	public function maybe_add_meta_via_pre_wp_44_method( $id, $meta ) {
-		if ( -1 !== version_compare( get_bloginfo( 'version' ), '4.4' ) ) {
+		if ( - 1 !== version_compare( get_bloginfo( 'version' ), '4.4' ) ) {
```
Comment (borkweb) [inline src/Tribe/Aggregator/Record/Abstract.php:649]: A space between `-` and a number only makes sense when you are writing an equation (e.g. `5 - 1`). When writing a single negative number, it should be written `-1`. To reinforce my statement, simply run `number_format( - 1 );` and you'll see that the result is a proper `-1` with no space.

```suggestion
		if ( -1 !== version_compare( get_bloginfo( 'version' ), '4.4' ) ) {
```

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 8]
```diff
@@ -991,7 +992,7 @@ public function query_child_records( $args = [] ) {
 	 *
 	 * @return WP_Query|WP_Error|bool
 	 */
-	public function get_child_record_by_status( $status = 'success', $qty = -1, array $args = [] ) {
+	public function get_child_record_by_status( $status = 'success', $qty = - 1, array $args = [] ) {
```
Comment (borkweb) [inline src/Tribe/Aggregator/Record/Abstract.php:995]:
```suggestion
	public function get_child_record_by_status( $status = 'success', $qty = -1, array $args = [] ) {
```

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 9]
```diff
@@ -1112,7 +1129,7 @@ public function is_schedule_time() {
 		$next    = $last + $this->frequency->interval;
 
 		// let's add some randomization of -5 to 0 minutes (this makes sure we don't push a schedule beyond when it should fire off)
-		$next += ( mt_rand( -5, 0 ) * 60 );
+		$next += ( mt_rand( - 5, 0 ) * 60 );
```
Comment (borkweb) [inline src/Tribe/Aggregator/Record/Abstract.php:1132]:
```suggestion
		$next += ( mt_rand( -5, 0 ) * 60 );
```

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 10]
```diff
@@ -1535,7 +1552,7 @@ public function insert_posts( $items = [] ) {
 			if ( $should_import_settings && isset( $event['sticky'] ) ) {
 				if ( $event['sticky'] == true ) {
 					$event['EventShowInCalendar'] = 'yes';
-					$event['menu_order']          = -1;
+					$event['menu_order']          = - 1;
```
Comment (borkweb) [inline src/Tribe/Aggregator/Record/Abstract.php:1555]:
```suggestion
					$event['menu_order']          = -1;
```

File: src/views/v2/widgets/widget-events-list.php [hunk 11]
```diff
@@ -53,11 +53,13 @@
 
 			<?php $this->template( 'components/data' ); ?>
 
+			<?php if ( ! empty( $widget_title ) ) : ?>
 			<header class="tribe-events-widget-events-list__header">
 				<h2 class="tribe-events-widget-events-list__header-title tribe-common-h6 tribe-common-h--alt">
 					<?php echo esc_html( $widget_title ); ?>
 				</h2>
 			</header>
```
Comment (borkweb) [inline src/views/v2/widgets/widget-events-list.php:61]: These should be tabbed in by one level, now.

**Review by borkweb — CHANGES_REQUESTED** [review]

File: src/views/v2/widgets/widget-events-list.php [hunk 12]
```diff
@@ -53,11 +53,13 @@
...
+			<?php endif; ?>
```
Comment (borkweb) [inline src/views/v2/widgets/widget-events-list.php:62]:
```suggestion
			<?php if ( ! empty( $widget_title ) ) : ?>
				<header class="tribe-events-widget-events-list__header">
					<h2 class="tribe-events-widget-events-list__header-title tribe-common-h6 tribe-common-h--alt">
						<?php echo esc_html( $widget_title ); ?>
					</h2>
				</header>
			<?php endif; ?>
```

**Pull request merged**